oslabs-beta/Kubiquity
# Add Navbar

WHAT DOES THIS PR DO? 
1. Adds a Navbar component to the app. 
2. Conditionally renders our components based on Navbar. 
3. Updates to file names. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 25 && npm install && npm run dev`
2. Verify you see buttons in our Navbar. 
3. Click the buttons, and verify they work as expected. Home should show all component; Metrics should show our memory graph; Log should show our table; About should show nothing, because we don't have an about component yet. 
4. Verify that app still works. 

ADDITIONAL NOTES: 
1. Styling not included for the Navbar—to come in future PR. 
2. I decided to do this based on conditional rendering rather than react router, because once we GET our data, we don't want to navigate away from our data, because that means we'd have to either refetch it OR introduce Redux.
